frontend/src/components/charts/AttemptSummaryChart.tsx

```diff
@@ -15,7 +23,7 @@ export const VerticalBarChart = () => {
         xAxis: {
           // name: "Difficulty",
           type: "category",
-          data: ["Easy", "Medium", "Hard"],
+          data: [Difficulty.EASY, Difficulty.MEDIUM, Difficulty.HARD],
```

Comment:
Should we only capitalise the first letter?

Reply:
I think its fine to go with all caps